gst/tensor_merge/gsttensormerge.c

```diff
@@ -441,6 +457,26 @@ gst_tensor_merge_collect_buffer (GstTensorMerge * tensor_merge,
 
   walk = tensor_merge->collect->data;
 
+  if (tensor_merge->need_set_time) {
+    while (walk) {
+      GstCollectData *data = (GstCollectData *) walk->data;
+      walk = g_slist_next (walk);
+
+      GstBuffer *buf = NULL;
+
+      buf = gst_collect_pads_peek (tensor_merge->collect, data);
```

Comment:
when getting buffer using gst_collect_pads_peek, should unref the buffer gst_buffer_unref(buf)

Reply:
Fixed. Thanks.